conf/carbon.conf.example

```diff
@@ -262,6 +262,14 @@ RELAY_METHOD = rules
 # datapoint to more than one machine.
 REPLICATION_FACTOR = 1
 
+# If your REPLICATION_FACTOR >=2, you are using more than one replica per server
```

Comment:
Not trying to be an oxford comma groupie here, but I think we need one at the end of this sentence or it takes on a different meaning.